server/admin/member.js

```diff
@@ -107,5 +128,9 @@ module.exports = {
     memberLeave: {
         endpoint: '/admin/member/leave',
         handler: createLeaveHandler
+    },
+    reap: {
+        endpoint: '/admin/reap',
+        handler: createReapHandler
```

Comment:
Also add a test for `/admin/reap`, with 3 cases:

1. trying to reap self.
2. trying to reap non-faulty.
3. trying to reap faulty.